ui/src/transaction.ts

```diff
@@ -136,36 +136,47 @@ export const formatStake = (msg: Message): string => `${formatMessage(msg)} depo
 export const formatPayer = (identity: identity.Identity): object => {
   return {
     name: identity.metadata.displayName || identity.metadata.handle,
-    variant: "user",
+    variant: "USER",
```

Comment:
```suggestion
    variant: SubjectVariant.User,
```

This should be the actual variant we define down below.

Reply:
Sweet, made a `Variant` type that can be used by both for now